services/storage/src/simcore_service_storage/s3_client.py

```diff
@@ -108,11 +108,14 @@ async def create(
             config=Config(signature_version="s3v4"),
         )
         assert isinstance(session_client, ClientCreatorContext)  # nosec
+        exit_stack = contextlib.AsyncExitStack()
         client = cast(S3Client, await exit_stack.enter_async_context(session_client))
         # NOTE: this triggers a botocore.exception.ClientError in case the connection is not made to the S3 backend
         await client.list_buckets()
 
-        return cls(session, client, s3_max_concurrency)
+        return cls(  # pylint: disable=too-many-function-args
+            client, session, exit_stack, s3_max_concurrency
+        )
```

Comment:
not sure I get the issue with pylint. But I have the impression there is something wrong with how you use the asyncexitstack here.

Reply:
do you even need to overload the create function?

Reply:
You are right I do not need to overload, will use the one from aws package. :+1: